common/models/user.js

```diff
@@ -693,25 +617,51 @@ module.exports = function(User) {
    * ```
    *
    * @options {Object} verifyOptions
-   * @property {String} type Must be 'email'.
+   * @property {String} type Must be `'email'` in the current implementation.
+   * @property {Function} mailer A mailer function with a static `.send() method.
+   *  It must accept the verifyOptions object, the method'sremoting context options
+   *  object and a callback function with `(err, {email, verificationToken, uid})`
```

Comment:
👍  you are totally right, i got wrong when reading the cb signature, swapping with the overall verify cb